### DISCUSSION TITLE
Feature request "Silent mode"

### DISCUSSION THREAD

**Discussion by AnAnalogGuy** [gcormier/megadesk#132] [Ideas]
"Toggle Audio Feedback" does only supress audio feedback on pushing the buttons (at least on my megadesk), audio feedback from triggering megadesk to move into a specific stored position remains regardless of the "Audio Feedback" setting. 

While audio feedback is very useful in many situations and for many users, others might rather like a silent mode (me in example :-)). I was wondering if it would be an option to add such a silent mode?

**Comment by gcormier** [comment]
Sure, opened #133
